### PR TITLE
Visual signal proposal

### PR DESCRIPTION
**Pull request by ben47955**

### CODE REVIEW

File: rules/forbidden_actions.tex [hunk 1]
```diff
@@ -112,13 +112,24 @@ \subsubsection{Penalty Area During Penalty Kick}
 
 If a robot enters the relevant penalty area during a penalty kick, except for the goalkeeper (defending team) and one robot of the attacking team, ``Illegal Position'' is called.
 
-\subsection{Motion in Set}
+\subsection{Forbidden Motion}
+Robots that begin moving their legs or locomote in any fashion before the specified state
```
Comment (arnemoos) [inline rules/forbidden_actions.tex:116]: Maybe add something like: 

>  before the specified state, see the subsections, will be penalized

File: rules/forbidden_actions.tex [hunk 2]
```diff
@@ -112,13 +112,24 @@ \subsubsection{Penalty Area During Penalty Kick}
...
+ will be penalized \textit{in place} on the field for \qty{15}{\second} in \textit{playing}.
+``Forbidden Motion'' penalties do not follow the standard removal procedure, and hence do not count towards the incremental penalty count.
+A robot will be moved back to its original position if it has moved significantly before becoming penalized.
+
+\subsubsection{Motion in Initial}
+\label{sec:motion_in_initial}
+
+Robots that begin moving before \texttt{ready} (\ie before the head referee does the visual signal).
```
Comment (arnemoos) [inline rules/forbidden_actions.tex:124]: > before the transition from \texttt{initial} to \texttt{ready}

File: rules/game_process.tex [hunk 3]
```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee will announce the start of the \texttt{ready} state for both halves by raising both hands covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in the a following years.}.
```
Comment (arnemoos) [inline rules/game_process.tex:78]: > The referee will announce the transition from the \texttt{initial} to \texttt{ready} state

> by raising both hands, covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in the a following years.}, straight over its head.

File: rules/game_process.tex [hunk 4]
```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
...
+The GameController \texttt{ready} signal will be delayed by \qty{\SetDelayTimeChampion}{\second} for champion cup and by \qty{\SetDelayTimeChallenger}{\second} for challenger shield.
```
Comment (arnemoos) [inline rules/game_process.tex:79]: > The GameController signal for the transition from \texttt{initial} to \texttt{ready} will be delayed

> champions cup and by \qty{\SetDelayTimeChallenger}{\second} for challenge shield

File: rules/game_process.tex [hunk 5]
```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee will announce the start of the \texttt{ready} state for both halves by raising both hands covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in the a following years.}.
+The GameController \texttt{ready} signal will be delayed by \qty{\SetDelayTimeChampion}{\second} for champion cup and by \qty{\SetDelayTimeChallenger}{\second} for challenger shield. 
+The referee must stand on the side of the field at halfway line level.
```
Comment (arnemoos) [inline rules/game_process.tex:80]: > The referee must stand on the opposite side of the game controller behind the touchline and at the height of the halfway line.

**Review by arnemoos — CHANGES_REQUESTED** [review]

**Comment by ben47955** [comment]
I made change to reflect the feedback. 
Also note the current proposal include a visual signal for return from timeout. This was intentional from my part. This was not discussed in the meeting, so feel free to express any concern.

**Review by arnemoos — APPROVED** [review]

File: rules/game_process.tex [hunk 6]
```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee will announce the transition from the \texttt{initial} to \texttt{ready} state for both halves by raising both hands, covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following years.}, straight over its head.
```
Comment (rvilling) [inline rules/game_process.tex:78]: Suggest rewording as:

The referee will announce the transition from the \texttt{initial} to \texttt{ready} state for both halves by raising both hands over their head.  The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following years.}.

Question: Approx. how long does the referee hold this pose? Are there any constraints on where the referee is positioned or oriented while giving this signal?

File: rules/game_process.tex [hunk 7]
```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
...
+The GameController signal for the transition from \texttt{initial} to \texttt{ready} will be delayed by \qty{\SetDelayTimeChampion}{\second} for champions cup and by \qty{\SetDelayTimeChallenge}{\second} for challenge shield.
```
Comment (rvilling) [inline rules/game_process.tex:79]: Minor edits: "for champions cup" -> "for the Champions Cup" and "for challenge shield" -> "for the Challenge Shield". 
I recommend searching for other occurrences - as these are proper nouns, they should be capitalized each time.

File: rules/game_process.tex [hunk 8]
```diff
@@ -75,6 +75,10 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
+The referee will announce the transition from the \texttt{initial} to \texttt{ready} state for both halves by raising both hands, covered with red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following years.}, straight over its head.
+The GameController signal for the transition from \texttt{initial} to \texttt{ready} will be delayed by \qty{\SetDelayTimeChampion}{\second} for champions cup and by \qty{\SetDelayTimeChallenge}{\second} for challenge shield. 
+The referee must stand on the opposite side of the game controller behind the touchline and at the height of the halfway line.
```
Comment (rvilling) [inline rules/game_process.tex:80]: This definition probably belongs earlier to understand the context of the signals. See my other comment above.

File: rules/game_process.tex (hunk 6 above)
Comment (ben47955) [reply rules/game_process.tex:78]: I was thinking to state to hold the position for two seconds. Considering robots should already looks at the referee. 
I want to make the detection of this signal realistic with the evolution of the rule. I can see a proposal for in game signal to take five seconds, from whistles to end of gestures.
